src/components/form/fields/identifierField.jsx

```diff
@@ -111,6 +111,7 @@ class IdentifierField extends React.Component {
                 value={value.raw || ''}
                 placeholder={placeholder}
                 title={label}
+                autoComplete={this.props.autoComplete}
```

Comment:
```suggestion
                autoComplete={autoComplete}
```
Ajoute `autoComplete` à la décomposition des props ligne 90 à 100.

Reply:
Est-ce que ça vaut vraiment un "require change" ?